ios/browser/api/url_sanitizer/url_sanitizer_service+private.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_IOS_BROWSER_API_URL_SANITIZER_URL_SANITIZER_SERVICE_PRIVATE_H_
+#define BRAVE_IOS_BROWSER_API_URL_SANITIZER_URL_SANITIZER_SERVICE_PRIVATE_H_
+
+#import <Foundation/Foundation.h>
+
+#include "base/memory/raw_ptr.h"
+#include "brave/components/url_sanitizer/browser/url_sanitizer_service.h"
+#include "brave/ios/browser/api/url_sanitizer/url_sanitizer_service.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface URLSanitizerService (Private)
+
+- (instancetype)initWithURLSanitizerService:
+    (raw_ptr<brave::URLSanitizerService>)urlSanitizer;
```

Comment:
raw_ref/raw_ptr should only be used for class fields, not method sigs. This should just return `brave::URLSanitizerService*`

Reply:
Done